### PR TITLE
Sunset auction feature

### PR DESCRIPTION
**Pull request by ismailToyran**
### Project organization
- Related [trello card](https://trello.com/c/ciwfL2rr/1139-sunset-auction-feature-final-steps)

### Description
Officially sunset the “Auction system” feature. Remove everything from the repository related to auction.
- One addition to the PR [Add status filter back with new addition Has offers](https://github.com/liteflow-labs/starter-kit/pull/497/commits/f6254e15e3a7a5f3bb735a00dc9335c17b8dfa94) to have multiple option for status filter.

### Checklist
- [x] Base branch of the PR is `dev`

### CODE REVIEW

File: components/Filter/FilterAsset.tsx [hunk 1]
```diff
@@ -166,45 +153,6 @@ const FilterAsset: NextPage<Props> = ({
               </AccordionPanel>
             </AccordionItem>
           ))}
-        <AccordionItem>
```
Comment (antho1404) [inline components/Filter/FilterAsset.tsx:169]: We still need a way to only display NFTs with listing offers.

File: hooks/useAssetFilterFromQuery.ts [hunk 2]
```diff
@@ -117,28 +108,6 @@ const maxPriceFilter = (
     } as AssetToManyOfferFilter,
   } as AssetFilter)
 
-const offersFilter = (offers: OfferFilterType, date: Date): AssetFilter => {
```
Comment (antho1404) [inline hooks/useAssetFilterFromQuery.ts:120]: We should still be able to filter listed assets

File: pages/users/[id]/on-sale.gql [hunk 3]
```diff
@@ -7,6 +7,7 @@ query FetchOnSaleAssets(
   $orderBy: [AssetsOrderBy!]
 ) {
   onSale: assets(
+    # TODO: find a new way to filter on sale assets after removal of auctions
```
Comment (antho1404) [inline pages/users/[id]/on-sale.gql:10]: TODO left here

**Review by antho1404 — COMMENTED** [review]
Pretty good, a lot of cleanup.
A few issues but it will be good to merge.
Let's keep in mind this should only be merged on Jan 1st

File: pages/users/[id]/on-sale.gql (hunk 3 above)
Comment (NicolasMahe) [reply pages/users/[id]/on-sale.gql:10]: You should be able to filter them like:
```
    filter: {
      quantity: { greaterThan: "0" }
      sales: {
        some: {
          makerAddress: { equalTo: $address }
          expiredAt: { greaterThan: $now }
        }
      }
    }
```

File: components/HomeSection/Assets.gql [hunk 4]
```diff
@@ -49,33 +49,6 @@ query FetchAssets(
...
-                name
-                id
-                decimals
-                symbol
-              }
-            }
-          }
-        }
-      }
       bestBid: bids(
```
[the remark this answers is not in the document]
Comment (ismailToyran) [reply components/HomeSection/Assets.gql:52]: We still use `bestBid` on cards (and other areas), if it's not on sale, we show the highest bid

image

File: pages/users/[id]/on-sale.gql (hunk 3 above)
Comment (ismailToyran) [reply pages/users/[id]/on-sale.gql]: [Change onSale query filter](https://github.com/liteflow-labs/starter-kit/pull/497/commits/f0b3a9807b42f94e843796d10e00c8a0fd6f297d)

File: hooks/useAssetFilterFromQuery.ts (hunk 2 above)
Comment (ismailToyran) [reply hooks/useAssetFilterFromQuery.ts:120]: Let me know about this commit please [Add status filter back with new addition Has offers](https://github.com/liteflow-labs/starter-kit/pull/497/commits/f6254e15e3a7a5f3bb735a00dc9335c17b8dfa94)

File: components/Filter/FilterAsset.tsx (hunk 1 above)
Comment (ismailToyran) [reply components/Filter/FilterAsset.tsx]: Let me know about this commit please [Add status filter back with new addition Has offers](https://github.com/liteflow-labs/starter-kit/pull/497/commits/f6254e15e3a7a5f3bb735a00dc9335c17b8dfa94)

**Review by antho1404 — APPROVED** [review]

**Pull request merged**